tests/datasets/test_fsdnoisy18k.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+
+from tests.test_utils import run_clip_tests
+
+from soundata import annotations
+from soundata.datasets import fsdnoisy18k
+
+TEST_DATA_HOME = "tests/resources/sound_datasets/fsdnoisy18k"
+
+
+def test_clip():
+    default_clipid = "17"
+    dataset = fsdnoisy18k.Dataset(TEST_DATA_HOME)
+    clip = dataset.clip(default_clipid)
+    expected_attributes = {
+        "audio_path": "tests/resources/sound_datasets/fsdnoisy18k/FSDnoisy18k.audio_train/17.wav",
+        "clip_id": "17",
+    }
+
+    expected_property_types = {
+        "audio": tuple,
+        "tags": annotations.Tags,
+        "split": str,
+        "aso_id": str,
+        "manually_verified": int,
+        "noisy_small": int,
+    }
+
+    run_clip_tests(clip, expected_attributes, expected_property_types)
+
+    default_clipid_test = "564"
+    clip_test = dataset.clip(default_clipid_test)
+    expected_attributes_test = {
+        "audio_path": "tests/resources/sound_datasets/fsdnoisy18k/FSDnoisy18k.audio_test/564.wav",
+        "clip_id": "564",
+    }
+
+    expected_property_types_test = {
+        "audio": tuple,
+        "tags": annotations.Tags,
+        "split": str,
+        "aso_id": str,
+        "manually_verified": None.__class__,
```

Comment:
Do you need the `__class__` here?

Reply:
If I test it without the `__class__` it does not work! For that reason I am using it. Do you know if there's a different way to test that an attribute is `None` here? Thanks!

Reply:
I see, you're right. It's due to the check we do in `test_utils` (` assert isinstance(getattr(clip, prop), expected_property_types[prop]`). 

we could change that line to check for `None` or maybe change `None.__class__` for `type(None)` which might be clearer? (the result is the same though). I'm ok either way

Reply:
Agree! Le's use `type(None)` it is way clearer! Thanks for the suggestion @magdalenafuentes. I guess the loader is ready to go now:)